src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -1105,6 +1149,72 @@ const schema = {
       type: 'boolean',
       default: false,
     },
+    mentalChanges: {
+      type: 'object',
+      properties: {
+        depression: {
+          type: 'boolean',
+        },
+        obsessive: {
+          type: 'boolean',
+        },
+        prescription: {
+          type: 'boolean',
+        },
+        substance: {
+          type: 'boolean',
+        },
+        hypervigilance: {
+          type: 'boolean',
+        },
+        agoraphobia: {
+          type: 'boolean',
+        },
+        fear: {
+          type: 'boolean',
+        },
+        other: {
+          type: 'boolean',
+        },
+        otherExplanation: {
+          type: 'string',
+        },
+        noneApply: {
+          type: 'boolean',
+        },
+      },
+    },
+    hospitalizationHistory: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          hospitalizationDateRange: {
+            $ref: '#/definitions/dateRange',
+          },
+          hospitalName: {
+            type: 'string',
+            minLength: 1,
+            maxLength: 100,
+          },
+          hospitalAddress: {
+            type: 'object',
+            properties: _.omit(
+              ['addressLine3', 'country'],
+              baseAddressDef.properties,
+            ),
+          },
+        },
+      },
+    },
+    ptsdIncidents: {
```

Comment:
We don't currently have a transformer to put ptsd incidents and secondary incidents into this shape, I'm making a new ticket for this.

Reply:
For clarity, you mean to transform the `incident${index}` (and friends) pages into a proper array because we couldn't use an array to generate pages in the first place, but the data really belongs in an array. Aye?

Reply:
Yessir, exactly that.